server/controllers/nodemailerController.js

```diff
@@ -1,26 +1,86 @@
 var nodemailer = require('../config/nodemailer');
 var GuestQuery = require('../queries/guestQueries');
+var EmailQuery = require('../queries/emailQueries');
+var EventQuery = require('../queries/eventQueries');
+var UserQuery = require('../queries/userQueries');
+var ItemQuery = require('../queries/itemQueries');
+
+// writes an emailbody base
+var createInviteManyBody = function (creator, event, url, callback) {
+  var body = "";
+  body += "<h5>Hey There!</h5>";
+  body += "<h4>" + creator + " has invited you to : </h4>";
+  body += "<a href='" + url + "'><h3>" + event.name + "!</h3></a>";
+  if(event.date !== null) {
+    body += "<p>When: <b>" + event.date + "</b></p>";
+  }
+  if(event.location !== null) {
+    body += "<p>Where: <b>" + event.location + "</b></p>";
+  }
+  body += "<h4>" + creator + " suggests we get the following:";
+  ItemQuery.getAll(event.id, function (items){
+    body += "<ul>";
+    items.forEach(function(item){
+      body += "<li>" + item.name + "</li>";
+    });
+    body += "</ul>";
+    body += "<a href='" + url +"'><h4> click here to sign up for items or to add more! </h5></a>";
+    callback(body);
+  });
+
+};
+
+var createInviteOneBody = function (event, mode) {
```

Comment:
are these empty functions necessary.
are they for future functionality or required for current feature